libs/GraphicECS/SFML/Components/SoundName.hpp

```diff
@@ -0,0 +1,34 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundName
+*/
+
+#ifndef SOUNDNAME_HPP_
+#define SOUNDNAME_HPP_
+
+#include "Component/Component.hpp"
+#include "GraphicECS/SFML/Resources/SoundResource.hpp"
+
+namespace ecs
+{
+    /// @brief SoundName store the key as enum to the corresponding sound stores in shared resource SoundResource.
+    class SoundName : public Component {
+      public:
+        /// @brief The sound name stores to find the corresponding soundName.
```

Comment:
The sound name stores an enum to find which sound it needs in SoundResources shared resource